### PR TITLE
Update billing FAQ [ work in progress ]

### PR DESCRIPTION
**Pull request by teesloane**
Getting started with moving new pricing structure into our FAQ. 

I'm not sure how much of the original billing section FAQ should stick around. Also, I did not add FAQ's for any of the question/answers in the google doc that had comments / anything we're unsure of. 

Also, I suppose some of the plan details regarding credits / payments per the three plans could be in a table rather than a list, so let me know if that would be useful.

### CODE REVIEW

File: jekyll/_cci2/faq.md [hunk 1]
```diff
@@ -248,6 +248,86 @@ Yes, the billing is associated with the organization. You can buy while within t
...
+### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+
+Every month, you are charged for your selected credit package at the beginning of the month.
+Bonus credits are offered on the Performance plan for 50,000 credit purchases and above.
+
+### What happens when I run out of credits?
+{:.no_toc}
+
+On the Seed plan, you never have to worry about compute spend going above your budget. Your projects will not build until your billing cycle is complete. You will get an error message in-app and through email explaining why your build did not run. If you decide you want to build more, you can upgrade to a Performance plan if you choose.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: > Your projects will not build until your billing cycle is complete.

That is confusing to me. I think of it as "Your projects will not build until your next billing cycle begins and your monthly credit package is refilled.

Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: > If you decide you want to build more, you can upgrade to a Performance plan if you choose.

They could also increase their credit package size for the Seed plan, correct?

File: jekyll/_cci2/faq.md [hunk 2]
```diff
@@ -248,6 +248,86 @@ Yes, the billing is associated with the organization. You can buy while within t
...
+### What happens to my credits if I upgrade to a new plan?
+{:.no_toc}
+
+If you upgrade within Seed, the rates are exactly the same.
+If you upgrade to Performance, credits rollover from your Seed plan.
+
+### Do credits expire?
+{:.no_toc}
+
+**Hobbyist**: Credits expire every week.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: Just want to make sure that we're aligned with Rose. We've been calling them "Free", "Basic", and "Performance". I want to make sure we're all on the same page with naming our plan tiers.

File: jekyll/_cci2/faq.md [hunk 3]
```diff
@@ -248,6 +248,86 @@ Yes, the billing is associated with the organization. You can buy while within t
...
+
+- The day you purchase a plan, you’ll get a charge for the credit plan you choose.
+- The end of your billing cycle, you’ll get a charge for any users above \$5.
+- On the second billing cycle, you’ll be charged for the credit plan chosen and
+  (if above 5 users), the previous user count.
+
+**Performance Plan**
+
+- The day you purchase a plan, you’ll be charged a one time charge for the number of credits you purchase.
+- You will be charged a recurring flat fee for 5 users. The following month, you will be charged this flat user fee plus a charge per user over 5 users from the previous billing period. For example, during January you used 7 users, you will be charged for 2 additional users at $15 each on February’s bill.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: This is not how we charge for User Seats. They are chosen and paid for up front. We block if they run over.

File: jekyll/_cci2/faq.md (hunk 2 above)
Comment (teesloane) [reply jekyll/_cci2/faq.md]: Will make the change unless noted otherwise.

File: jekyll/_cci2/faq.md (hunk 1 above)
Comment (bostonaholic) [reply jekyll/_cci2/faq.md]: That makes sense. Maybe:

> Your projects will not build until your next billing cycle begins and your recurring credit package is refilled

File: jekyll/_cci2/faq.md (hunk 3 above)
Comment (bostonaholic) [reply jekyll/_cci2/faq.md]: Yes, that looks correct.

File: jekyll/_cci2/faq.md [hunk 4]
```diff
@@ -248,6 +250,66 @@ Yes, the billing is associated with the organization. You can buy while within t
...
+
+The following FAQ questions are for our __Credit Usage Plans_.
+
+### How do the new pricing plans affect me as a customer?
+{:.no_toc}
+For the vast majority of customers, you can keep your current plan for now and this simply represents a new option you may want to consider. 
+
+### What are credits?
+{:.no_toc}
+Credits are used to pay for your usage based on machine type and size.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: Credits will also be used to pay for features like Docker Layer Caching.

File: jekyll/_cci2/faq.md [hunk 5]
```diff
@@ -248,6 +250,66 @@ Yes, the billing is associated with the organization. You can buy while within t
...
+{:.no_toc}
+You pay to the nearest credit. First we round up to the nearest second, and then up to the nearest credit. On the 2x4 (medium) box this is effectively in 6 second increments. 
+
+### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+### What happens when I run out of credits?
+{:.no_toc}
+On the Performance plan, your credit package will be refilled at 10% of your monthly selected package size when you have 5% remaining on your credit package so builds will never be blocked on a lack of credits in your account.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: We don't refill _at_ 10%, we refill _exactly_ 10% of your monthly selection package size when you reach 5% remaining. If your monthly package size is 25,000 credits, you will automatically be refilled 2,500 credits when you reach 1,250 remaining credits.

File: jekyll/_cci2/faq.md (hunk 1 above)
Comment (teesloane) [reply jekyll/_cci2/faq.md]: In the google doc we've removed all content for everything but the `performance` plan, afaict

File: jekyll/_cci2/faq.md (hunk 3 above)
Comment (rosaliebud) [reply jekyll/_cci2/faq.md]: What does this mean? Why does it wait until $500 is exceeded?

> - Anytime your team’s balance exceeds $500 from adding new users.

File: jekyll/_cci2/faq.md [hunk 6]
```diff
@@ -230,24 +230,96 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
...
+
+#### If a container is used for under one minute, do I have to pay for a full minute? 
+{:.no_toc}
+You pay to the next nearest credit. First we round up to the nearest second, and then up to the nearest credit.
+
+#### How do I buy credits? Can I buy in any increments?
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+#### How do I buy credits? Can I buy in any increments?
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: Looks like a duplicate section from above.

File: jekyll/_cci2/faq.md [hunk 7]
```diff
@@ -230,24 +230,96 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
...
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+#### What do I pay for?
+{:.no_toc}
+You can choose to pay for premium features per active user, compute, and optionally, premium support.
+
+- Access to features, such as new machine sizes, and Docker Layer Caching (DLS) are paid with a monthly fee of $15 per active user.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: > Docker Layer Caching (DLC)

Comment (bostonaholic) [reply jekyll/_cci2/faq.md]: DLC is paid for with credits per usage, similar to compute credits. It is not paid for via the $15 per active user charge.

File: jekyll/_cci2/faq.md [hunk 8]
```diff
@@ -230,24 +230,96 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
...
+{:.no_toc}
+**Performance Plan**: Credits expire one year after purchase. Unused credits will be forfeited when the account subscription is canceled.
+
+#### How do I pay?
+{:.no_toc}
+You can pay from inside the CircleCI app for monthly pricing.
+
+#### When do I pay?
+{:.no_toc}
+On the Usage Plans, you will be charged for the plan selections you’ve made for credits, user seats, and premium support tiers at the beginning of every billing cycle.
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: Credit refills are paid for at the moment the refill occurs.

File: jekyll/_cci2/faq.md [hunk 9]
```diff
@@ -230,24 +230,94 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
...
+{:.no_toc}
+Every month, you are charged for your selected credit package at the beginning of the month.
+
+#### What do I pay for?
+{:.no_toc}
+You can choose to pay for premium features per active user, compute, and optionally, premium support.
+
+- Access to features, such as new machine sizes, are paid with a monthly fee of $15 per active user. 
+- Compute is paid for monthly in credits for the machine size and duration you use.
+- Docker Layer Caching (DLS) is paid for with credits per usage, similar to
```
Comment (bostonaholic) [inline jekyll/_cci2/faq.md]: > Docker Layer Caching (DLC)

File: jekyll/_cci2/faq.md [hunk 10]
```diff
@@ -230,24 +230,95 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
...
+- Docker Layer Caching (DLC) is paid for with credits per usage, similar to
+  compute credits.
+
+#### What happens when I run out of credits?
+{:.no_toc}
+
+On the Performance plan, when you reach 5% of your remaining credits, you will
+be refilled 10% of your credits. For example, If your monthly package size is 25,000 credits, you will automatically be refilled 2,500 credits when you reach 1,250 remaining credits.
+
+#### What happens to my credits if I upgrade to a new plan?
```
Comment (rosaliebud) [inline jekyll/_cci2/faq.md]: We should not include this questions for now

File: jekyll/_cci2/faq.md [hunk 11]
```diff
@@ -230,24 +230,95 @@ CircleCI provides no guarantees about precision. A scheduled workflow will be ru
...
+
+#### What are the other renewal dates?
+{:.no_toc}
+
+The first credit card charge on the day you upgrade to a paid plan or change paid plans, in addition to the following charges from CircleCI:
+
+- On the monthly renewal date if your team is on the monthly plan.
+- On the annual renewal date if your team is on the annual plan.
+- On the last day of the month if your team is on the annual plan and there is an outstanding balance from adding new users or utilizing more credits.
+- Anytime your team’s balance exceeds $500 from adding new users.
```
Comment (rosaliebud) [inline jekyll/_cci2/faq.md]: Remove this line

**Pull request merged**